MaterialEntryControl.md

```diff
@@ -4,24 +4,418 @@ Text fields let users enter and edit text.
 [View Material Design documentation](https://material.io/components/text-fields)
 
 ## Screenshot
+<!-- TODO: Improve this  -->
 <img src="https://github.com/HorusSoftwareUY/MaterialDesignControlsPlugin/blob/master/screenshots/entry.gif" width="300">
 
 ## Example MaterialEntry
+Set style:
+
+```XML
+<Style TargetType="material3:MaterialEntry"
```

Comment:
Maybe could be more clear in this readme sample if we use in the styles fixed values (xamarin font sizes, xamarin colors or hexadecimal colors) instead of static resources.